docs/how_to/menus.rst

```diff
@@ -109,9 +109,9 @@ root. But if you want the menu to be attached to a CMS Page you can do that as
 well.
 
 Instead of extending from :class:`~menus.base.Menu` you need to extend from
```

Comment:
I've not see this notation in RST that includes a tilde before. Its probably totally legit, but thought I'd point it out in case its a old typo.